examples/rbd/fs-pod-rx.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-fs-rx-volume
+  labels:
+    app: pod-fs-rx-volume
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: pod-fs-rx-volume
+  template:
+    metadata:
+      labels:
+        app: pod-fs-rx-volume
+    spec:
+      containers:
+        - name: web-server
+          image: docker.io/library/nginx:latest
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: mypvc
+              mountPath: /var/lib/www/html
+      volumes:
+        - name: mypvc
```

Comment:
This may not work when the pods run on different nodes, do not include that as an example as it might confuse users trying it out.

Reply:
Yeah, could you also please suggest a better location for these files?

Reply:
Rename the file to `fs-pod-rox.yaml` and make the appropriate changes? Including a ROX example is fine.

Reply:
Yeah, good idea :+1: